client/src/components/header.tsx

```
@@ -26,259 +42,487 @@
+
+  const handleClose = () => {
+    setDisplayName(initialDisplayName);
+    setSelectedImage(initialImageUrl);
+    setImageUrl(initialImageUrl);
+    setOpen(false);
+  };
+
+  const handleImageChange = async (
+    event: React.ChangeEvent<HTMLInputElement>,
+  ) => {
+    if (event.target.files && event.target.files[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)